### ISSUE TITLE
run-tests failures

### ISSUE THREAD

**Issue by Ygrex** [aidanholm/luakit#406]
building 47b4088f1382d2e2d94f1d0310a8109910d01dee with:
```
$ make DEVELOPMENT_PATHS=1 LUA_PKG_NAME=lua5.1 clean run-tests
```
results in the following tests failures:
1. test_module_binds_have_descriptions
```
FAIL async/test_module_binds_have_descriptions / test_module_binds_have_descriptions
  tests/async/test_module_binds_have_descriptions.lua:70: Some bindings are missing descriptions:
  lib/binds.lua                                          Description for binding :reload doesn't end in a full stop.
                                                         Description for binding :tabde[tach] doesn't end in a full stop.
  Traceback:
  (1) tests/async/test_module_binds_have_descriptions.lua:70 in function [anonymous]
```
2. test_undo_close_works
```
FAIL async/test_undoclose / test_undo_close_works
  Timed out while waiting for signal 'load-status'
    interval was 500msec
    
  Traceback:
  (1) tests/lib.lua:26                  in function wait_for_view
  (2) tests/async/test_undoclose.lua:22 in function [anonymous]
```
3. test_undo_close_restores_tab_history
```
FAIL async/test_undoclose / test_undo_close_restores_tab_history
  tests/async/test_undoclose.lua:66: Expected objects to be equal.
  Passed in:
  (number) 2
  Expected:
  (number) 3
  Traceback:
  (1) /usr/share/lua/5.1/luassert/assert.lua:51 in function is_equal
  (2) tests/async/test_undoclose.lua:66         in function [anonymous]
```
4. test_website_data
```
FAIL async/test_clib_luakit / test_website_data
  Timed out while waiting
    interval was 200msec
    
  Traceback:
  (1) tests/async/test_clib_luakit.lua:104 in function [anonymous]
```

**Comment by aidanholm** [comment]
Thanks for the fix! Could you make the PR against https://github.com/luakit/luakit instead please?

By the way, I'm mostly working on https://github.com/luakit/luakit now, and will stop pushing to this repo soon.

**Comment by aidanholm** [comment]
Fixed merged on main repository at 7846b4ff
